common/compositor/va/varenderer.cpp

```diff
@@ -294,6 +352,43 @@ bool VARenderer::Draw(const MediaState& state, NativeSurface* surface) {
   param.filters = nullptr;
   param.filter_flags = VA_FRAME_PICTURE;
 
+  VAProcFilterCapColorBalance vacaps[VAProcColorBalanceCount];
+  uint32_t vacaps_num = VAProcColorBalanceCount;
+
+  if (caps_.empty()) {
+    if (!QueryVAProcFilterCaps(
+           va_context, VAProcFilterColorBalance, vacaps, &vacaps_num))
+      return false;
+    else
+        SetVAProcFilterDefaultValue(&vacaps[0]);
+  }
+
+  std::array<ScopedVABufferID, 5> cb_elements({va_display_, va_display_,
+                                               va_display_, va_display_,
+                                               va_display_});
+  std::vector<VABufferID> filters;
+  VAProcFilterParameterBufferColorBalance cbparam;
+  cbparam.type = VAProcFilterColorBalance;
+  cbparam.attrib = VAProcColorBalanceNone;
+
+  for (ColorBalanceCapMapItr itr = caps_.begin(); itr != caps_.end(); itr++) {
+    if (itr->second.value != itr->second.default_value) {
```

Comment:
you can't compare  float value

Reply:
yes, I will use another way to fix this.